api/src/controllers/team.js

```diff
@@ -68,11 +71,11 @@ async function storeHandle(req, res, next) {
 }
 
 /**
- * Gets the team info
- * @param {*} req request object contains the teamId decoded in auth middleware
- * @param {*} res response object, the team related info
- * @returns 200: the team related info
- */
+  * Gets the team info
+  * @param {*} req request object contains the teamId decoded in auth middleware
+  * @param {*} res response object, the team related info
+  * @returns 200: the team related info
+  */
```

Comment:
please fix spaces, ditto for other docs

Reply:
the thing is in the actual file it just has one space each but in the git preview it shows two spaces, can you confirm if its 1 spaced by opening it on VS code?